src/vs/workbench/parts/files/browser/views/openEditorsViewer.ts

```diff
@@ -375,6 +375,11 @@ export class ActionProvider extends ContributableActionProvider {
 							revertAction.setResource(resource);
 							revertAction.enabled = openEditor.isDirty();
 							result.push(revertAction);
+
+							const showModificationsAction = this.instantiationService.createInstance(ShowModificationsAction, ShowModificationsAction.ID, ShowModificationsAction.LABEL);
```

Comment:
Maybe rename this to "Compare with Saved" to make it clear what it does? And then we could maybe move this command into the place where today we have "Select for Compare"?

![image](https://user-images.githubusercontent.com/900690/27944046-564a8180-62e4-11e7-9bb9-49ebd573b712.png)